src/GafferSceneUI/LightTool.cpp

```diff
@@ -2371,6 +2485,239 @@ class CornerHandle : public LightToolHandle
 		std::optional<PlanarDrag> m_drag;
 };
 
+class RadiusHandle : public LightToolHandle
+{
+	public :
+		RadiusHandle(
+			const std::string &lightType,
+			SceneView *view,
+			const InternedString &radiusParameter,
+			const float radiusToHandleRatio,
+			const bool faceCamera,
+			const bool useScale,
+			const std::string &name = "RadiusHandle"
+		) :
+			LightToolHandle( lightType, view, {radiusParameter}, name ),
+			m_radiusParameter( radiusParameter ),
+			m_radiusToHandleRatio( radiusToHandleRatio ),
+			m_faceCamera( faceCamera ),
+			m_useScale( useScale ),
+			m_dragDirection()
+		{
+		}
+
+		/// \todo In the constructor above, accidentally omitting the `useScale` argument and
+		/// passing a string will implicitly convert it to bool, potentially causing
+		/// `faceCamera` to be unintentially true with no compiler warnings.
+		/// Is there a better way to prevent that than this deleted constructor?
+		RadiusHandle(
+			const std::string &lightType,
+			SceneView *view,
+			const InternedString &radiusParameter,
+			const bool faceCamera,
+			const char *name
+		) = delete;
```

Comment:
Since this is an internal API, I wouldn't worry too much. But can't we just remove the default value from `name`?

Reply:
I took out the default `name` and the deleted constructor in https://github.com/GafferHQ/gaffer/pull/5483/commits/804500fcc57257623619203dc3d7933592b55375, https://github.com/GafferHQ/gaffer/pull/5483/commits/43007ad85a32fe0ae7415fd4d7f09cbff2eed17a and https://github.com/GafferHQ/gaffer/pull/5483/commits/e5bd74cece3322bc91da6fdffa5904f6e01d02a1. I hadn't done it before out of a desire to fit in with the majority of other nodes and things that give a default name. But, as in life, sometimes fitting in is not something to aspire to.